src/sass/partials/_footer.scss

```diff
@@ -0,0 +1,47 @@
+.footer {
+  font-family: $font-primary;
+  font-size: $font-size-footer-mobile;
+  font-weight: $font-weight-regular;
+  background-color: $footer-color-background; //#F7F7F7
+  color: $text-color-additional-secondary; //#545454
+  text-align: center;
+  opacity: 0.95;
+  padding: 10px;
+  position: sticky;
+  bottom: 0;
+  display: flex;
+
+  @media screen and (min-width: $breakpoint-tablet) {
+    font-size: $font-size-footer;
+    opacity: 1;
+    padding: 30px;
+    position: relative;
+    bottom: auto;
+  }
+}
+
+.footer > p {
```

Comment:
Dont use style rules like this because if we put another paragraph inside footer its gonna be complicated. Try to style always with classes.